### ISSUE TITLE
Deploy via Windows 8 / 8.1 Client Hyper-V

### ISSUE THREAD

**Issue by znmeb** [boot2docker/boot2docker#641]
Enhancement request - I'd like to use Docker with a Linux guest running on the "native" Windows virtualizer, Client Hyper-V, rather than VirtualBox. How hard is this?

**Comment by SvenDowideit** [comment]
yup. however, to be clear, implementing this should not be called __only__ - I started work on it, but it needs admin rights, which many don't have, and worse, on OSX turned out to be unreliable.

https://github.com/boot2docker/boot2docker-cli/pull/247 is where I was experimenting with it before we shipped the vbox shares -

**Comment by blaggacao** [comment]
OSX couldn't use nfs instead? - I would best-guess so.
For my understanding:
On windows, there would be need for a dedicated user, right? - this is why admin rights would be required (one-time), not quite so in order to modify the file permissions. Or did I miss something?

**Comment by SvenDowideit** [comment]
yup, iirc, i had some weird errors or corner cases to deal with on NFS and OSX, and of course, you have to get root access every time

and the same on windows - you don't need to make a user, you need to create a share, which is also a privileged operation.

Mind you, I was also trying to avoid sharing `/Users`...

and avoid sharing that share on the network (ie restricted to the virtual host only network)

its all in the details :)

**Comment by blaggacao** [comment]
;-D Well on the long run it's probably worth it, as this was some PITA in order to get sourcecode from my windows IDE onto docker and back and forth and, .... well it's still a mess ;)
